scripts/assign-colors

```diff
@@ -14,7 +14,7 @@ if __name__ == '__main__':
 
     parser.add_argument('--ordering', type=str, required=True,
         help="""Input TSV file defining the color ordering where the first
-        column is the field and the second column is the trait in that field.
+        column is the category and the second column is the trait in that category.
```

Comment:
[sidebar] I always struggle to name these things, especially when they have a value attached. Category / trait / value seems pretty good.